homeassistant/components/bluecurrent/manifest.json

```
@@ -0,0 +1,13 @@
+{
+  "domain": "bluecurrent",
+  "name": "BlueCurrent",
```

According to their website, it is written as `Blue Current`. Their entry at the chamber of commerce is also "Blue Current B.V.".

```suggestion
  "name": "Blue Current",
```